### ISSUE TITLE
LightGBM Rank Model predict score is negative

### ISSUE THREAD

**Issue by Man1978-scd** [microsoft/LightGBM#6295]
When I do a rank task, I use the lightgbm ranker model, the loss function uses lambdarank, the data labels are 0 and 1, representing the correlation, when I do the prediction, the return result is negative, I don't quite understand why the score is negative. When we do binary classification, the score returned by the model can be understood as a probability, does anyone know what that means? thank you.

**Comment by shiyu1994** [comment]
Hi @Man1978-scd, thanks for using LightGBM. Which API are you using with LightGBM, and how is the prediction done? I think in your case LightGBM is providing the raw prediction scores.

Also. not that the goal of lambdarank (or the underlying algorithm lambdamart) is not to predict the relevance scores of the documents in a query. Instead, the goal is to provide a ranking order of the documents as consistent as the ordering according to their relevance scores.

So, though the predicted scores are not direct regression to the original relevance scores, but their values do indicate the relevance. A higher score indicates a higher relevance of a document.

Thus, negative values are allowed, since the lambdarank cares only about the ranking, not the actual score value.

**Comment by shiyu1994** [comment]
You may refer to this document for a more clear understanding of the LambdaMart algorithm.
https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/MSR-TR-2010-82.pdf
